lib/databases/mysql

```diff
@@ -16,7 +16,8 @@ MYSQL_DRIVER=${MYSQL_DRIVER:-PyMySQL}
 register_database mysql
 
 # Linux distros, thank you for being incredibly consistent
-MYSQL=mysql
+#MYSQL=mysql
+MYSQL=mariadb
```

Comment:
add a check to the line below to do this

Reply:
Yes, it should be covered by is_clearlinux().